teradatalabs/mapr52-hive/files/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
Hmm, yea but for example `setup.sh` for `cdh5-hive-master` does a bunch of things, so it's OK to name it something generic like `setup.sh` whereas here all we're doing is configuring MySQL so I think we should rename it. Don't feel strongly about it though. I can also see the argument for being consistent with other image files. Up to you.